server/app/views/applicant/NorthStarApplicantProgramSummaryView.java

```diff
@@ -0,0 +1,113 @@
+package views.applicant;
+
+import auth.CiviFormProfile;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import controllers.AssetsFinder;
+import controllers.applicant.ApplicantRoutes;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import modules.ThymeleafModule;
+import org.thymeleaf.TemplateEngine;
+import play.mvc.Http.Request;
+import services.applicant.Block;
+
+/** Renders a list of sections in the form with their status. */
+public final class NorthStarApplicantProgramSummaryView extends NorthStarApplicantBaseView {
+
+  @Inject
+  NorthStarApplicantProgramSummaryView(
```

Comment:
question(non-blocking):  Why make the constructor package-private?

Reply:
I think it's generally preferable to have @Inject constructors package private instead of public to prevent another class from trying to manually invoke the constuctor